scripts/vm-guest-tools.bat

```diff
@@ -22,7 +22,7 @@ if not exist "C:\Windows\Temp\windows.iso" (
 )
 
 cmd /c ""C:\Program Files\7-Zip\7z.exe" x "C:\Windows\Temp\windows.iso" -oC:\Windows\Temp\VMWare"
-cmd /c C:\Windows\Temp\VMWare\setup.exe /S /v"/qn REBOOT=R\"
+cmd /c C:\Windows\Temp\VMWare\setup64.exe /S /v"/qn REBOOT=R\"
```

Comment:
This is likely PEBKAC. I'll try it again with setup.exe. It was likely due to the partial installs up to that point having run the enable-winrm script multiple times to progress the provisioning.